hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusUpdateTopicHandler.java

```diff
@@ -157,6 +148,25 @@ public void handle(@NonNull final HandleContext handleContext) {
         topicStore.put(builder.build());
     }
 
+    @NonNull
+    @Override
+    public Fees calculateFees(@NonNull final FeeContext feeContext) {
+        requireNonNull(feeContext);
+        final var op = feeContext.body();
+        final var topicUpdate = op.consensusUpdateTopicOrThrow();
+        final var topicId = topicUpdate.topicIDOrElse(TopicID.DEFAULT);
+        final var topic = feeContext.readableStore(ReadableTopicStore.class).getTopic(topicId);
+
+        return feeContext.feeCalculator(SubType.DEFAULT).legacyCalculate(sigValueObj -> {
+            try {
+                return new UpdateTopicResourceUsage()
+                        .usageGivenExplicit(fromPbj(op), sigValueObj, topic != null ? pbjToState(topic) : null);
```

Comment:
I think we don't need this `topic != null ? pbjToState(topic) : null` since `usageGivenExplicit` already checks of topic is null